src/services/TestResultsService.ts

```diff
@@ -350,16 +350,16 @@ export class TestResultsService {
         .then((testResults) => {
           const filterTestTypes: any[] = [];
           testResults.forEach((testResult: { testTypes: any; vehicleType: any; vehicleSize: any; vehicleConfiguration: any; noOfAxles: any; }) => {
-            testResult.testTypes.forEach((testType: { testTypeClassification: string; }) => {
-              if (testType.testTypeClassification === TEST_TYPE_CLASSIFICATION.ANNUAL_WITH_CERTIFICATE) {
+            testResult.testTypes.forEach((testType: { testExpiryDate: string; }) => {
+              if (testType.testExpiryDate) {
                 filterTestTypes.push(testType);
               }
             });
           });
           return filterTestTypes;
         }).then((testTypes) => {
           testTypes.forEach((testType) => {
-            if (dateFns.isAfter(testType.testExpiryDate, maxDate) && testType.testTypeClassification === TEST_TYPE_CLASSIFICATION.ANNUAL_WITH_CERTIFICATE) {
+            if (dateFns.isAfter(testType.testExpiryDate, maxDate) && testType.testExpiryDate) {
```

Comment:
second condition is redundant - it's a condition of the test type making it past the previous step.

Reply:
Agree with Kevin